thirdweb/modules/nft.py

```diff
@@ -256,8 +290,10 @@ def set_restricted_transfer(self, restricted: bool = False):
         """
 
         tx = self.__abi_module.set_restricted_transfer.build_transaction(
-            restricted, self.get_transact_opts())
+            restricted, self.get_transact_opts()
+        )
         self.execute_tx(tx)
 
     def get_abi_module(self) -> NFT:
+        """To be removed"""
```

Comment:
Lets remove this comment, these methods need to stay for the module to function correctly.

Reply:
Makes sense, changes made.